opencti-platform/opencti-graphql/src/domain/stixObjectOrStixRelationship.ts

```diff
@@ -1,19 +1,21 @@
 import { elLoadById } from '../database/engine';
-import { READ_PLATFORM_INDICES } from '../database/utils';
+import { READ_PLATFORM_INDICES, UPDATE_OPERATION_ADD } from '../database/utils';
 import { storeLoadById } from '../database/middleware-loader';
 import { ABSTRACT_STIX_REF_RELATIONSHIP } from '../schema/general';
 import { FunctionalError } from '../config/errors';
 import { isStixRefRelationship } from '../schema/stixRefRelationship';
-import { createRelation, createRelations, deleteRelationsByFromAndTo } from '../database/middleware';
+import { createRelations, deleteRelationsByFromAndTo, patchAttribute } from '../database/middleware';
 import { notify } from '../database/redis';
 import { BUS_TOPICS } from '../config/conf';
 import type { AuthContext, AuthUser } from '../types/user';
 import type { StixRefRelationshipAddInput, StixRefRelationshipsAddInput } from '../generated/graphql';
+import type { BasicStoreObject, StoreMarkingDefinition } from '../types/store';
+import { schemaRelationsRefDefinition } from '../schema/schema-relationsRef';
 
 type BusTopicsKeyType = keyof typeof BUS_TOPICS;
 
-export const findById = async (context: AuthContext, user: AuthUser, id: string) => {
-  return elLoadById(context, user, id, { indices: READ_PLATFORM_INDICES });
+export const findById = async <T extends BasicStoreObject> (context: AuthContext, user: AuthUser, id: string) : Promise<T> => {
```

Comment:
Duplicate of middleware-loader.internalLoadById ?

Reply:
These changes are for adding type for now and not handling this :)

Reply:
It's better to don't change it at all in this case :)
Because, it's not normal to return an await here

You should have

export const findById = async (context: AuthContext, user: AuthUser, id: string) => {
  return internalLoadById(context, user, id, { indices: READ_PLATFORM_INDICES });
};